spec/fixtures/CommonBuildSettings/Project/Objc_OSX_Native/ViewController.m

```diff
@@ -3,15 +3,17 @@
 @implementation ViewController
 
 - (void)viewDidLoad {
-    [super viewDidLoad];
+	[super viewDidLoad];
```

Comment:
Ah, my Xcode settings seem to use tabs, but that doesn't really matter here does it? (basically unused code)